src/services/auth/plugins/password/password.schemas.ts

```diff
@@ -20,9 +21,17 @@ export const passwordLogin = {
   querystring: customType.StrictObject({
     lang: Type.Optional(Type.String()),
   }),
+  response: {
+    [StatusCodes.OK]: customType.StrictObject({
+      resource: Type.String({ description: 'Redirection link' }),
+    }),
```

Comment:
We provide the link as text ? not with a redirect ?

Reply:
With axios, you cannot redirect with a post request. 
Perhaps we should get rid of axios, as redirect also returns as errors... but it would be a much bigger reshape.